app/controllers/favorites_controller.rb

```diff
@@ -11,11 +11,17 @@ def update
     redirect_to "/pets/#{params[:pet_id]}"
   end
 
-  def destroy
+  def delete
```

Comment:
changed this to delete and set the new method to destroy because that felt more accurate - delete removes only a single one, while destroy tears down the whole set of favorites